server/config.ts

```diff
@@ -45,7 +45,7 @@ export default {
   },
   apis: {
     audit: {
-      queueUrl: get('AUDIT_QUEUE_URL', 'eu-west-2', requiredInProduction),
+      queueUrl: get('AUDIT_SQS_QUEUE_URL', 'eu-west-2', requiredInProduction),
       region: get('AUDIT_QUEUE_REGION', 'http://localhost:4566/000000000000/mainQueue', requiredInProduction),
```

Comment:
there is a bit of transposition gone on here:
```suggestion
      region: get('AUDIT_QUEUE_REGION', 'eu-west-2', requiredInProduction),
```